cmd/prerequisites/prerequisites.go

```diff
@@ -0,0 +1,18 @@
+package prerequisites
+
+import (
+	"github.com/armosec/armo-platform-tools/poc-prerequisite/kubescape-sizing-checker/pkg/sizingchecker"
```

Comment:
@matthyx  should be changed to https://github.com/kubescape/sizing-checker/tree/main/pkg/sizingchecker